mapper/mapper.go

```diff
@@ -470,6 +481,8 @@ func getIDNameLookup(vals dataset.Options) map[string]string {
 func CreateHierarchyPage(h hierarchyClient.Model, parents []hierarchyClient.Parent, dst dataset.Model, f filter.Model, curPath, dimensionTitle, datasetID, releaseDate string) hierarchy.Page {
 	var p hierarchy.Page
 
+	log.Debug("mapping api response models to hierarchy page", log.Data{"filterID": f.FilterID, "datasetID": datasetID})
```

Comment:
dimension title would potentially also be useful here, if the dataset contains more than one possible hierarchy, knowing which one/if all are throwing errors would be useful